build/Linux+BSD/mscore.appdata.xml.in

```diff
@@ -0,0 +1,316 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2017 Juraj Fiala <jurf_AT_riseup.net> -->
```

Comment:
According to CLA, we have to put Werner Schweer name to any copyright mentions. You can replace this with text like
`Copyright 2018 Werner Schweer and others. Maintainer of the file is Juraj Fiala <jurf_AT_riseup.net>.

Reply:
Sure, will do.